frontend/src/renderer/util/autoSortStandards.svelte

```diff
@@ -104,17 +115,32 @@
         imageStack.bestDarkFieldImages = [bestDarkFieldImage, maxBy(images, probabilityScoreProperties[3])];
         images = filter(images, (e) => e !== maxBy(images, probabilityScoreProperties[3]))
 
+        // Uses string similarity to determine which images go into which columns
+        const sortImageByLighting = function (image1, image2, exampleImageA) {
+            if(findBestMatch(exampleImageA.name, [image1.name, image2.name]).bestMatchIndex === 0) {
+                return [image1, image2];
+            }
+            return [image2, image1];
+        };
```

Comment:
This is how images are sorted by A-B. We compare both of each type (say both darkField images) to a known 'A' image. Whichever one is more similar gets put in the same column

Reply:
Is this code able to be updated with user preferences down the line?  I feel like using a delimiter could be better if/when users can add their own auto-sort terms.